lib/tasks/gemcutter.rake

```diff
@@ -79,4 +79,24 @@ namespace :gemcutter do
       puts "Done."
     end
   end
+
+  namespace :rubygems_version do
+    desc "Backfill gem versions with rubygems_version."
+    task backfill: :environment do
+      without_rubygems_version = Version.where(rubygems_version: nil)
+      mod = ENV['shard']
```

Comment:
This is good to leave. It allows us to run the backfill in multiple parallel tasks.